helm/values.yaml

```
@@ -0,0 +1,46 @@
+maelstro_frontend:
+  enabled: true
+  replicaCount: 1
+  image:
+    repository: test
+  tolerations: []
+  affinity: {}
+  extra_environment: []
```

Use camelCase: https://helm.sh/docs/chart_best_practices/values/#naming-conventions

Good example: https://github.com/mviewer/helm-charts/blob/main/mviewer/values.yaml#L27

---

I prefer to be close to : https://github.com/georchestra/helm-georchestra/blob/main/values.yaml#L20

---

It's history, it's not a good example. All values need to be camel case, it's written in the helm guide: https://helm.sh/docs/chart_best_practices/values/#naming-conventions.

And it's a work that I'm trying to fix in all our helm charts.

Please use this convention instead:

```suggestion
  extraEnv: []
```

It's what all helm chart do. Bitnami included: https://github.com/bitnami/charts/blob/main/bitnami/postgresql/values.yaml#L402